src/github/client/github-client.ts

```diff
@@ -31,15 +32,32 @@ export default class GitHubClient {
 	private readonly githubInstallationId: InstallationId;
 	private readonly logger: Logger;
 
-	constructor(
+	public static async build(githubInstallationId: InstallationId,
```

Comment:
Our client is not a long-living entity

We create a new client instance when:
1. We process SQS Message
2. Cann get-github-configuration REST API

That mean the timeout will be active only within a scope of a small operation, which usually takes less than a minute
Hence we don't care if our feature flag change won't be applied immediately.

Reply:
@KAbakumovAtl Oh.  That's not good.  The whole point of having a client is that it's long lived.  It can better manage connections or have the ability to abort old connections.  It could also potentially create a memory leak if you just keep creating it or at the very least the GC will keep getting hammered.